adapter/src/main/java/org/mifos/chatbot/adapter/handler/loan/ArrearDayHandler.java

```diff
@@ -35,13 +36,14 @@
 @Slf4j
 @Component
 public class ArrearDayHandler extends BaseLoanIntentHandler {
-    private static final String[] INTENT_KEYWORDS = {"arrear", "day"};
+    private static final String[] INTENT_KEYWORDS = {"arrear_day"};
```

Comment:
Move to static file

Reply:
This INTENT_KEYWORDS is specific for this class, so my intention is to keep it here or should i move it?

Reply:
Keep it here then.